howtos/run_conan_in_docker.rst

```diff
@@ -0,0 +1,181 @@
+.. _docker_conan:
+
+How to run Conan with Docker
+============================
+
+With Docker, you can run different virtual Linux operating systems in a Linux, Mac OSX or Windows machine.
+It is useful to reproduce build environments, for example to automate CI processes. You can have
+different images with different compilers or toolchains and run containers every time is needed.
+
+We provide :ref:`list of pre-built images<available_docker_images>` with common
+build tools and compilers and of course, with Conan installed.
+
+
+Using conan inside a container
+------------------------------
+
+.. code-block:: bash
+
+    docker run -it --rm lasote/conangcc7 /bin/bash
+
+The previous code will run a shell in container. We have specified:
+
+- ``-it``: Keep STDIN open and allocate a pseudo-tty, in other words, we want to type in the container because we are opening a bash.
+- ``--rm``: Once the container exits, remove the container. Helps to keep clean or hard drive.
+- ``lasote/conangcc7``: Image name, check the :ref:`available docker images<available_docker_images>`.
+- ``/bin/bash``: The command to run
+
+
+Now we are running on the conangcc7 container we can use Conan normally. It is always recommended
+to upgrade conan from pip first:
+
+.. code-block:: bash
+
+    $ sudo pip install conan --upgrade
+    $ git clone https://github.com/conan-community/conan-openssl
+    $ cd conan-openssl
```

Comment:
Clarify what you are doing here: "We are creating a package from the recipe, for OpenSSL; etc"